NEWS.md

```diff
@@ -28,7 +28,7 @@ New language features
   are now interpreted as `BigInt` ([#23546]).
 * The postfix conjugate transpose operator `'` now accepts Unicode modifiers as
   suffixes, so e.g. `a'ᵀ` is parsed as `var"'ᵀ"(a)`, which can be defined by the
-  user. `a'ᵀ` parsed as `a' * ᵀ` before, so this is a minor change ([#37247]).
+  user. `a'ᵀ` parsed as `a' * ᵀ` before, so this is a minor change ([#38062]).
```

Comment:
No, #37247 is definitely the new language feature talked about here. #38062 just added `'ᵀ` as a new standard library function and is already mentioned below. I did mess up the link for that though.

Reply:
Right, as I said, I made that comment by accident. Can you merge the other and close the issue? Or get someone to do it?

Reply:
I feel a bit uncomfortable merging commits manually into master. Usually the best solution to problems like this is for the author of the PR to do an [interactive rebase](https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History) on top of `HEAD~2`, where you drop the first commit. You did allow maintainers to push to your branch, so if you get stuck and can't figure it out, myself or someone else with commit rights could probably do it for you.

Reply:
I've seen individual commits picked so I just thought it was no problem. And while I do know about rebase, have done it locally, this time I only made the PR through Github, so it will have to wait. You have my permission to rebase or pick the commit. I'll leave the conversation "unresolved", so that people do not merge both by accident. Unless you want me to resolve and maybe move to a Draft.